modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/io/DDMFormJSONSerializer.java

```diff
@@ -66,9 +67,31 @@ public DDMFormSerializerSerializeResponse serialize(
 				ddmForm.getDefinitionSchemaVersion());
 		}
 
+		List<DDMFormField> ddmFormFields = ddmForm.getDDMFormFields();
+
+		for (DDMFormField ddmFormField : ddmFormFields) {
```

Comment:
Hi @pavel-savinov , Can you please sort this? Just put 
`_addMissingLocales(
				ddmFormField.getPredefinedValue(),
				ddmForm.getAvailableLocales());`
between 
`...ddmFormField.getLabel...` and `...ddmFormField.getTips()...`